src/formatter.rs

```diff
@@ -175,6 +175,7 @@ mod tests {
     use super::*;
 
     #[test]
+    #[allow(clippy::cognitive_complexity)]
```

Comment:
We are linting cognitive_complexity in tests?

Reply:
clippy does